Makefile.kokkos

```diff
@@ -491,6 +487,7 @@ ifeq ($(KOKKOS_INTERNAL_USE_SYCL), 1)
 endif
 
 ifeq ($(KOKKOS_INTERNAL_USE_OPENMPTARGET), 1)
+  KOKKOS_LIBS += -latomic
```

Comment:
Please confirm that we dont get to guard for `KOKKOS_INTERNAL_ENABLE_DESUL_ATOMICS` equal to 1

Reply:
Yeah, I am considering getting rid of that option anyway.